db/Controllers/topics.controller.js

```diff
@@ -0,0 +1,17 @@
+
+const { fetchTopics } = require("../Models/models")
+
+const getTopics = (request, response, next)=>{
+
+    fetchTopics()
+    .then((topics)=>{ response.status(200)
+        .send({topics})
+     /* <--- passes err to app.js next? */
+    }).catch((err)=>{
+        console.log(err)
```

Comment:
Do you want console.log(err) here or do you want to make use of the `next` function which you have in your function parameters on line 4 to invoke the next middleware chain?